app/plugin_api.go

```diff
@@ -107,6 +107,10 @@ func (api *PluginAPI) GetTeamByName(name string) (*model.Team, *model.AppError)
 	return api.app.GetTeamByName(name)
 }
 
+func (api *PluginAPI) GetTeamsUnreadForUser(userId, teamIdToExclude string) ([]*model.TeamUnread, *model.AppError) {
+	return api.app.GetTeamsUnreadForUser(userId, teamIdToExclude)
```

Comment:
I wonder if we remove the `teamIdToExclude` parameter to keep the plugin API simpler. It's included in the REST API since it's a specific case needed by the client but I don't think that case really makes sense for a plugin

Thoughts @chclaus @hanzei @lieut-data ?

Reply:
Makes sense.

Reply:
👍

Reply:
Hmmm...

I'm not finally sure if it's a good idea to make the plugin API more sparse than the REST-API. I could imagine that there are some people who could port some own applications to plugins and might be appreciate if they find the same API.

But if you all think it's a good idea to change the signature, I'll do this this evening CET. ;)